Dockerfile.testing

```diff
@@ -0,0 +1,172 @@
+# vim: set ft=dockerfile:
+
+##------------------------------------------------------------------------------
+FROM artifactory.reecenet.org:6555/python:3.6 AS py36
```

Comment:
Given this is public, should we maybe not be Artifactory proxying?

Reply:
Ah yeah, forgot to remove those

I was thinking of having a script that used `sed` to put them back in if we were testing locally .. but I dunno.

Reply:
I think the advantages of using an Artifactory proxy are so minimal it's not really worth it, but up to you.

Reply:
fixed